ods_ci/tests/Tests/1300__model_registry/1302_model_registry_model_serving.robot

```diff
@@ -0,0 +1,277 @@
+${DC_S3_TYPE}=                       Object storage
+${NAMESPACE_ISTIO}=                  istio-system
+${NAMESPACE_MODEL-REGISTRY}=         odh-model-registries
+${SECRET_PART_NAME_1}=               modelregistry-sample-rest
+${SECRET_PART_NAME_2}=               modelregistry-sample-grpc
+${SECRET_PART_NAME_3}=               model-registry-db
+
+
+*** Test Cases ***
+Verify Model Registry Integration With Secured-DB
+    [Documentation]    Verifies the Integartion of Model Registry operator with Jupyter Notebook
+    [Tags]    OpenDataHub
+    Create Workbench    workbench_title=${WORKBENCH_TITLE}    workbench_description=Registry test
+    ...                 prj_title=${PRJ_TITLE}    image_name=Minimal Python  deployment_size=Small
+    ...                 storage=Persistent   pv_existent=${NONE}
+    ...                 pv_name=${NONE}  pv_description=${NONE}  pv_size=${NONE}
+    Workbench Should Be Listed      workbench_title=registry-wb
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE}
+    ${workbenches}=    Create List    ${WORKBENCH_TITLE}
+    Create S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=${DC_S3_NAME}
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=${AWS_BUCKET}    connected_workbench=${workbenches}
+    Data Connection Should Be Listed    name=${DC_S3_NAME}    type=${DC_S3_TYPE}    connected_workbench=${workbenches}
+    Open Data Science Project Details Page       project_title=${prj_title}    tab_id=workbenches
+    Wait Until Workbench Is Started     workbench_title=registry-wb
+    Upload File In The Workbench     filepath=${SAMPLE_ONNX_MODEL}    workbench_title=${WORKBENCH_TITLE}
+    ...         workbench_namespace=${PRJ_TITLE}
+    Upload File In The Workbench     filepath=${JUPYTER_NOTEBOOK_FILEPATH}    workbench_title=${WORKBENCH_TITLE}
+    ...         workbench_namespace=${PRJ_TITLE}
+    Launch And Access Workbench    workbench_title=${WORKBENCH_TITLE}
+    ...    username=${TEST_USER.USERNAME}     password=${TEST_USER.PASSWORD}
+    ...    auth_type=${TEST_USER.AUTH_TYPE}
+    Upload Certificate To Jupyter Notebook    ${CERTS_DIRECTORY}/domain.crt
+    Jupyter Notebook Should Run Successfully     ${JUPYTER_NOTEBOOK}
+
+
+*** Keywords ***
+Prepare Model Registry Test Setup
+    [Documentation]    Suite setup steps for testing Model Registry.
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    Launch Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    ...    ${ODH_DASHBOARD_URL}    ${BROWSER.NAME}    ${BROWSER.OPTIONS}
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Create Project
+    Apply ServiceMeshMember Configuration
+    Run OC Commands And Capture Output
```

Comment:
this kw is a bit weird, maybe update the name to specify what it's actually doing with oc?

Reply:
Already done that based on Berto's comment.